components/sensor/bme68x_bsec2.rst

```diff
@@ -0,0 +1,274 @@
+    consumption, sampling every 3 seconds or ``ULP`` for ultra-low power, sampling every 5 minutes.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **humidity** (*Optional*): Configuration for the humidity sensor.
+
+  - **sample_rate** (*Optional*, string): Optional sample rate override for this sensor. Can be ``LP`` for low power
+    consumption, sampling every 3 seconds or ``ULP`` for ultra-low power, sampling every 5 minutes.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+- **gas_resistance** (*Optional*): Configuration for the gas sensor.
+
+  - All options from :ref:`Sensor <config-sensor>`.
+
+- **iaq** (*Optional*): Configuration for the IAQ sensor.
+
+  - All options from :ref:`Sensor <config-sensor>`.
+
+- **iaq_static** (*Optional*): Configuration for the IAQ static sensor.
+
+  - All options from :ref:`Sensor <config-sensor>`.
+
+- **iaq_accuracy** (*Optional*): Configuration for the numeric IAQ accuracy sensor.
+
+  - All options from :ref:`Sensor <config-sensor>`.
+
+- **co2_equivalent** (*Optional*): Configuration for the CO₂ equivalent sensor.
+
+  - All options from :ref:`Sensor <config-sensor>`.
+
+- **breath_voc_equivalent** (*Optional*): Configuration for the Breath VOC equivalent humidity sensor.
+
+  - All options from :ref:`Sensor <config-sensor>`.
+
+Text Sensor
+-----------
+
+The sensor's accuracy can be reported in text format.
+
+Configuration variables:
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **bme68x_bsec2_i2c_id** (*Optional*, :ref:`config-id`): The ID of the ``bme68x_bsec2_i2c`` component the text sensor
+  will refer to. Useful when multiple devices are present in your configuration.
+- **iaq_accuracy** (*Optional*): Configuration for the IAQ accuracy sensor. Shows: ``Stabilizing``, ``Uncertain``,
+  ``Calibrating``, ``Calibrated``.
+
+  - All other options from :ref:`Text Sensor <config-text_sensor>`.
+
+Index for Air Quality (IAQ) Measurement
+---------------------------------------
+
+The measurements are expressed with an index scale ranging from 0 to 500. The index itself is deduced from tests using
+ethanol gas, as well as important VOC in the exhaled breath of healthy humans. The VOC values themselves are derived
+from several publications on breath analysis studies.  The BSEC2 software library defines the levels as follows:
+
++-----------+---------------------+
+| IAQ Index |    Air Quality      |
++===========+=====================+
+|  0 - 50   | Excellent           |
++-----------+---------------------+
+| 51 - 100  | Good                |
++-----------+---------------------+
+| 101 - 150 | Lightly polluted    |
++-----------+---------------------+
+| 151 - 200 | Moderately polluted |
++-----------+---------------------+
+| 201 - 250 | Heavily polluted    |
++-----------+---------------------+
+| 251 - 350 | Severely polluted   |
++-----------+---------------------+
+|   > 351   | Extremely polluted  |
++-----------+---------------------+
+
```

Comment:
```suggestion

This can be represented by a template text sensor such as below

.. code-block:: yaml

    text_sensor:
      - platform: template
        name: "BME68x IAQ Classification"
        lambda: |-
          if ( int(id(iaq).state) <= 50) {
            return {"Excellent"};
          }
          else if (int(id(iaq).state) >= 51 && int(id(iaq).state) <= 100) {
            return {"Good"};
          }
          else if (int(id(iaq).state) >= 101 && int(id(iaq).state) <= 150) {
            return {"Lightly polluted"};
          }
          else if (int(id(iaq).state) >= 151 && int(id(iaq).state) <= 200) {
            return {"Moderately polluted"};
          }
          else if (int(id(iaq).state) >= 201 && int(id(iaq).state) <= 250) {
            return {"Heavily polluted"};
          }
          else if (int(id(iaq).state) >= 251 && int(id(iaq).state) <= 350) {
            return {"Severely polluted"};
          }
          else if (int(id(iaq).state) >= 351) {
            return {"Extremely polluted"};
          }
          else {
            return {"error"};
          }
```